docs/source/getting-started/content/con_understanding-codeready-containers.adoc

```diff
@@ -12,4 +14,5 @@
 * It uses a single node which behaves both as a master and as a worker node.
 * The `machine-config`, `marketplace` and `monitoring` Operators are disabled by default.
 * These disabled Operators will cause the corresponding parts of the web console to be non functional.
+* For the same reason, there is currently no upgrade path to newer OpenShift versions.
```

Comment:
in general, we do not have an upgrade strategy (yet). due to the expiring certificates we have to make people aware of the ephemeral nature; else we also have to take care of moving images out and in to the new updated image?